app/assets/stylesheets/components/form/_form.scss

```diff
@@ -5,3 +5,18 @@
 .form__input-container--medium {
   max-width: 140px;
 }
+
+%form-row-2up {
```

Comment:
@benbarnett We've removed this styling for now — is this PR good to merge otherwise?